src/price/AbstractPrice.php

```diff
@@ -146,47 +146,20 @@ public function calculateSum(QuantityInterface $quantity): ?Money
         return $sum;
     }
 
+    /**
+     * What purpose of this method? Because it looks like duplicate of PriceHydrator::extract()
+     * Where we are using the result of this method?
+     * Magic calls can't be determined and I don't know what can be broken if we change the method result.
+     * Which structure must have the result, because array can contain anything?
```

Comment:
> What purpose of this method? Because it looks like duplicate of PriceHydrator::extract()

PriceHydrator::extract() is highly-bound to HiAPI context and public API. This method provides a naive way to encode all the properties.

> Where we are using the result of this method?

I can not remember the use cases. Maybe is not used at all not.

Add a logging statement with the stacktace, run tests and see when this method is being called.